### ISSUE TITLE
Adding to GNU/NonGNU ELPA

### ISSUE THREAD

**Issue by phikal** [anticomputer/age.el#2]
Hi,

would you be interested in having age.el added to one of the ELPAs?

**Comment by anticomputer** [comment]
Yeah once I have a stable first release out I'll do the work required to get it into one of the ELPAs or at the very least MELPA. I want to run it through its paces for a few weeks to make sure I've covered all the corner cases in daily use.

**Comment by minad** [comment]
Any news here? It would be great to see age.el on GNU/NonGNU ELPA. Thank you!

cc @monnier

**Comment by monnier** [comment]
I just added it to NonGNU ELPA.

It would be nice to move it to GNU ELPA, since I think we'll eventually want to merge that code into `auto-encryption-mode`, so I hope @anticomputer signs the copyright paperwork (AFAICT all the other contributions are too small to matter).

**Comment by anticomputer** [comment]
> I just added it to NonGNU ELPA.
> 
> It would be nice to move it to GNU ELPA, since I think we'll eventually want to merge that code into `auto-encryption-mode`, so I hope [@anticomputer](https://github.com/anticomputer) signs the copyright paperwork (AFAICT all the other contributions are too small to matter).

I'm happy to sign whatever copyright paperwork, but not sure it's needed since this is essentially just a port of code that is already FSF assigned? Not sure what the specifics are there, but if you can point me at the right thing I'm happy to take whatever steps needed to make age.el more broadly available/usable for folks.

**Comment by minad** [comment]
@anticomputer Thank you for writing this and @monnier thank you for adding this so quickly! Happy new year!
